front/app/containers/IdeasShow/index.tsx

```diff
@@ -213,28 +209,19 @@ const Content = ({
 
           <ProposedBudget ideaId={ideaId} projectId={project.id} />
 
-          <Box mb="40px">
+          <Box mb={compact ? '12px' : '40px'}>
             <Body
               postType="idea"
               postId={ideaId}
               body={ideaBody}
               translateButtonClicked={translateButtonIsClicked}
             />
           </Box>
-          {compact && participationContext && ideaIsInParticipationContext && (
-            <Box mb="16px">
-              {votingMethodConfig?.getIdeaPageVoteInput({
-                ideaId,
-                compact: true,
-                participationContext,
-              })}
-            </Box>
-          )}
           {compact &&
             participationContext?.attributes.participation_method !==
               'voting' &&
```

Comment:
Do you know why we actually not show this when PM = voting? 🤔 Would be useful if we add comments when implementing such exceptions.

Reply:
I'll add a comment so it's more clear :+1: It's not shown for voting in an effort to reduce bias by knowing who the author was who may have submitted a certain option.